### PR TITLE
#14 hobbies page

### PR DESCRIPTION
**Pull request by T1To48**
Mission Done

### CODE REVIEW

File: client/vite.config.js
Comment (T1To48) [inline client/vite.config.js:1]: add the host access in the here , becasue i run test on other pc that has windos os on it , also on iphone to see the ui ux in its final enivironment

**Review by yishain11 — CHANGES_REQUESTED** [review]
text should be bigger - your is around 9px and in figma its 12px. it should be easy to read

**Comment by T1To48** [comment]
> text should be bigger - your is around 9px and in figma its 12px. it should be easy to read. title - choose five intrests - is not in the same style as in figma
DONE!!

**Comment by T1To48** [comment]
WAITING FOR REVIEW

File: client/src/pages/register/Intrests.page.jsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
+import React, { useState, useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+import { interestsList } from "../../data/interest.js";
+import { useDispatch, useSelector } from "react-redux";
+import { addDetail } from "../../features/userRegister/userRegisterSlice.jsx";
+import { Navbar } from "../../components/styles/Navbar.jsx";
```
Comment (yishain11) [inline client/src/pages/register/Intrests.page.jsx:6]: create index for all the styles, and import all styled components in one import

File: client/src/pages/register/Intrests.page.jsx [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+    }
+    return setDisableSaveBtn(true);
+  }, [interestsNumber]);
+
+  return (
+    <div style={wrapper}>
+      <Navbar>
+        <Back>{"<"}</Back>
+        <PageTitle>Add Interests</PageTitle>
+        <EmptyDiv></EmptyDiv>
```
Comment (yishain11) [inline client/src/pages/register/Intrests.page.jsx:81]: what is the point of this?

File: client/src/pages/register/Intrests.page.jsx [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+              key={interest}
+              onClick={() => removeInterest(interest)}
+            >
+              {interest}
+              <span> X</span>
+            </button>
+          ))}
+        </div>
+
+        <div>
```
Comment (yishain11) [inline client/src/pages/register/Intrests.page.jsx:116]: export to another component

**Review by yishain11 — CHANGES_REQUESTED** [review]
client/src/pages/register/Intrests.page.jsx is too long. we wnat 70 lines of code.

File: client/src/pages/register/Intrests.page.jsx (hunk 2 above)
Comment (T1To48) [reply client/src/pages/register/Intrests.page.jsx:81]: this is the backarrow if the user wanna go back to the previous page
like shown in figma

**Review by yishain11 — CHANGES_REQUESTED** [review]
the design @ interests page is not the same as in figma - @ the header of the page.

**Pull request merged**